unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarAnimatorLegacy.cs

```diff
@@ -302,12 +290,28 @@ public bool Prepare(string bodyshapeId, GameObject container)
         animation = armatureParent.gameObject.GetOrCreateComponent<Animation>();
         armatureParent.gameObject.GetOrCreateComponent<StickerAnimationListener>();
 
-        BindBodyShape(animation, bodyshapeId, target);
+        PrepareLocomotionAnims(bodyshapeId);
+        SetIdleFrame();
+        animation.Sample();
         InitializeAvatarAudioAndParticleHandlers(animation);
         return true;
     }
 
-    public void PlayExpression(string expressionId, long timestamps) { SetExpressionValues(expressionId, timestamps); }
+    public void PlayEmote(string emoteId, long timestamps) { SetExpressionValues(emoteId, timestamps); }
```

Comment:
There's an ongoing confusion organization-wide in which we are calling all the animations as emotes. The emotes concept comes with the burden of being a fire and forget animation that is tied with the emotes UI, marketplace, etc. However, here we are indirectly calling "emotes" to the locomotion animations that do not have this burden and are not tied to any UI or marketplace.

Maybe we can call those methods `EquipAnimation` or `BindAnimation` to appease this concern?